setup/hr_family/odoo/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
@feketemihai Actually, if a module is merged without setup, the setup will be automatically generated but it's better to generate it directly.